data/bajra-technologies.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Bajra Technologies Pvt. Ltd.",
+  "establishedAt": 1293819300,
+  "location": "Dillibazar, Kathmandu, Nepal",
```

Comment:
```suggestion
  "location": "Dillibazar, Kathmandu",
```

Reply:
Will update same in other files as well @thenishantsapkota 🙏